src/cli/changed-cache.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+const crypto = require("crypto");
+
+// Generates a hash of the input string.
+function hash(data) {
+  return crypto
+    .createHash("sha1")
+    .update(data)
+    .digest("base64");
```

Comment:
Use `md4` (better use `xxHash`, but `xxHash` doesn't built-in in node :disappointed: ), `sha1` is very slow, also we don't need strong here

Reply:
We had a bit of a discussion about this on the issue and ended up finding that hashing is not a bottleneck in this case. I'm open to changing it, but I'm not sure it's worth adding a dependency.

https://github.com/prettier/prettier/issues/5853#issuecomment-469463458

Reply:
Okey, let's wait other feedback, for me `md4` is best solution here, when you have really big codebase, it is increase perf around 5%